vwifi.c

```diff
@@ -1579,10 +1579,15 @@ static int vwifi_stop_ap(struct wiphy *wiphy, struct net_device *ndev)
 
     return 0;
 }
-#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 7, 0)
+
 static int vwifi_change_beacon(struct wiphy *wiphy,
                                struct net_device *ndev,
-                               struct cfg80211_beacon_data *info)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 7, 0)
```

Comment:
Move the code fitting recent kernel versions earlier. That is,
```c
#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 7, 0)
    struct cfg80211_ap_update *info
#else
    struct cfg80211_beacon_data *info
#endif
```

Reply:
Does this need to be modified?
```c
#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 7, 0)
            .scan_width = NL80211_BSS_CHAN_WIDTH_20,
#endif
```

Reply:
> Does this need to be modified?

Yes, make it consistent.

Reply:
Later, we might introduce version check macro later. See https://github.com/sysprog21/simplefs/blob/master/simplefs.h#L78-L81